aws/resource_aws_lakeformation_datalake_settings.go

```diff
@@ -0,0 +1,136 @@
+package aws
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lakeformation"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+)
+
+func resourceAwsLakeFormationDataLakeSettings() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceAwsLakeFormationDataLakeSettingsPut,
+		Update: resourceAwsLakeFormationDataLakeSettingsPut,
+		Read:   resourceAwsLakeFormationDataLakeSettingsRead,
+		Delete: resourceAwsLakeFormationDataLakeSettingsReset,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"catalog_id": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Optional: true,
+				Computed: true,
+			},
+			"admins": {
+				Type:     schema.TypeList,
```

Comment:
I suggest 

```hcl
Type:     schema.TypeSet,
Set:      schema.HashString,
```

here if the order is not important.

Reply:
Indeed. Done.